apps/explorer/.env.example

```diff
@@ -0,0 +1,16 @@
+README.md# Enables mock mode (default = true)
+MOCK=true
+
+# Enables autoconnect for mock mode (default = true)
+AUTOCONNECT=true
+
+# Public IPFS gateway
+REACT_APP_IPFS_READ_URI=https://cloudflare-ipfs.com/ipfs
```

Comment:
probably we don't need the REACT_APP now?

Reply:
Did you test this work?

Reply:
Yes, it still in use for app-data decoding